weatherApp.js

```diff
@@ -206,25 +216,27 @@ const createHourBoxes = (day) => {
 
 const renderDivlist = (htmlElements, nameOfList) => {
 	const divList = document.createElement("div");
-       htmlElements.forEach((element) =>{
-           divList.appendChild(element);
-       });
+	htmlElements.forEach((element) =>{
+	   divList.appendChild(element);
+	});
     divList.className = nameOfList;
     const list = document.getElementsByClassName(nameOfList)[0];
     list.parentNode.replaceChild(divList, list);
 }
 
 let fetchWeather = (city)=>{
-/*    const requestUrl = `https://api.openweathermap.org/data/2.5/forecast?q=${city}&APPID=26c42f8b8305b74545f93ce538ad2356`;*/
-	const requestUrl = `dupadupadupadupadupadupadupa`;
+    const requestUrl = `https://api.openweathermap.org/data/2.5/forecast?q=${city}&APPID=26c42f8b8305b74545f93ce538ad2356`;
+	//const requestUrl = `dupadupadupadupadupadupadupa`;
     return fetch(requestUrl, {
         method: 'get'
     }).then((response)=> {
         return response.json(); //fetch zwraca obiekt response, a potem w wykonujemy funkcję json zawartą w prototypie tego response, żeby dostać potrzebne dane. json() tylko je wyciąga, a ich konkwersja dzieje się pod spodem
     }).catch(error => {
-		console.log("mockedData", mockedData);
+		console.log("error", error)
```

Comment:
For errors, use console.error

Reply:
Like this?
console.error("error", error);
If yes, it's done.

Reply:
Yes 👍